rust/cache/src/foyer.rs

```diff
@@ -328,6 +342,19 @@ where
             }
         };
 
+        let builder = match config.deterministic_hashing {
+            true => {
+                let rs = RandomState::with_seeds(
```

Comment:
generated from a good RNG

Reply:
Nit: add a comment in the code above to that effect?

Reply:
yep